org.liri.liri.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "org.liri.liri",
```

Comment:
liri.org redirects to https://www.domaination.com/lander/liri.org?domain=liri.org&redirect=ono-redirect

Did you see the PR checklist?

Reply:
I though it was to know if I controlled the github repository, not the domain.
No, I don't own (and don't plan) the liri.org domain. What do I need to update?

Reply:
Also:

![image](https://github.com/flathub/flathub/assets/114441/28810f00-494b-4c42-8577-0b04294caaa7)

So basically you can't use `org.li_ri` either.

Reply:
(previous comment deleted)

Use something related to your github.

Reply:
> I though it was to know if I controlled the github repository, not the domain.

No. It says:

>  The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). 

Is it unclear?

You just said you did NOT intend to have `liri.org`.

Reply:
Eg. `io.github.username.Li-Ri`


> What do I need to update?

ids in appdata, icon, desktop file, if you use the id anywhere in code etc. Then the id here in the manifest and filename.